### PR TITLE
@broskoski: Upgrades to Babel 6

### PR DESCRIPTION
**Pull request by dzucconi**
Closes https://github.com/artsy/metaphysics/issues/12

### CODE REVIEW

File: schema/artist/index.js [hunk 1]
```diff
@@ -18,9 +20,6 @@ import {
 let ArtistType = new GraphQLObjectType({
   name: 'Artist',
   fields: () => {
-    let Artwork = require('../artwork').default;
-    let PartnerShow = require('../partner_show');
-
```
Comment (dzucconi) [inline schema/artist/index.js]: No more deferred requires needed. Looks like the circular dependencies issue was fixed in 6.

**Comment by dzucconi** [comment]
cc @mzikherman + whoeever else. Make sure you `npm install` after syncing with upstream.

**Comment by broskoski** [comment]
:+1:

**Comment by mzikherman** [comment]
cool beans

**Pull request merged**